src/vigenere.rs

```diff
@@ -0,0 +1,91 @@
+// Implements http://rosettacode.org/wiki/Vigen%C3%A8re_cipher
+#![feature(convert)]
```

Comment:
Could you remove this feature usage? We are aiming at the Rust 1.0 release, which doesn't have it IIRC. You can replace all usages of `enc.as_str()` by `&enc`, which will automatically coerce `String` to `&str`.